test/integration/runner_common.cpp

```diff
@@ -75,16 +78,17 @@ int main(int argc, char* argv[]) {
         // directory is changed
         oidExe = fs::absolute(optarg);
         break;
-      case '\0':
-        extra_feature_args.push_back(std::string("-f") + optarg);
-        break;
       case 'h':
       default:
         usage(argv[0]);
         return 0;
     }
   }
 
+  if (const char* envArgs = std::getenv("OID_TEST_ARGS")) {
```

Comment:
Could we keep the arg processing too? Should be easy to make the latter one an append and have both. I find the command line stuff easier for manual use on a single test.